snapcraft/commands/lifecycle.py

```diff
@@ -296,3 +296,16 @@ class CleanCommand(_LifecycleStepCommand):
         remove the managed snap packing environment (VM or container).
         """
     )
+
+
+class TryCommand(_LifecycleCommand):
```

Comment:
Are all `_LifecycleCommand` options applicable to the `try` command? I suppose `--destructive-mode` works because in this case the `prime` directory is created directly on the host and there's no need no expose it?

Reply:
> Are all _LifecycleCommand options applicable to the try command? 

I think so, because `try` really is `prime + expose the prime dir` right? So any parameter that applies to `prime` could apply to `try` too. I can narrow it down if you think this is too broad though